### DISCUSSION TITLE
예시) DTO로 처리하기 vs 도메인으로 처리하기

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/jwp-chess#279] [토론]
요청을 보낼 때 HTTP Request의 Body로 보내는 값을 컨트롤러 메서드의 파라미터 객체로 받을 때
DTO를 만들어서 Body값을 받는게 좋을까요?
도메인 클래스로 Body값을 받는게 좋을까요?

만약 상황에 따라 다르다면 그 기준은 어떻게 정하면 좋을까요?

**Comment by bperhaps** [comment]
이 토론 이어가도 되는건가요? 중간곰이랑 계속 얘기했던 주제였는데......
